synapse/lib/types.py

```diff
@@ -812,11 +813,17 @@ async def _normPyDict(self, valu):
             mesg = f'No values provided for form {form.full}'
             raise s_exc.BadTypeValu(mesg=mesg)
 
-        norms = await self._normProps(form, valu)
+        if view is None:
+            view = False
```

Comment:
I think you can leave view as None here.

Reply:
I have it using false just so any nested gutors can skip attempting to grab the view from scope since it won't be present